tests/apps/demultiplex/conftest.py

```diff
@@ -322,3 +323,29 @@ def novaseq6000_flow_cell_sample_before_adapt_indexes() -> FlowCellSampleBcl2Fas
         SampleName="814206",
         Project="814206",
     )
+
+
+@pytest.fixture
+def bcl_convert_sample_sheet_path():
+    return Path(
+        "tests",
+        "fixtures",
+        "apps",
+        "demultiplexing",
+        "demultiplexed-runs",
```

Comment:
There is a fixture for this path called `demultiplexed_runs`
```suggestion
def bcl_convert_sample_sheet_path(demultiplexed_runs: Path):
    return Path(
        demultiplexed-runs,
```